op-service/txplan/txplan.go

```diff
@@ -384,6 +388,29 @@ func WithChainID(cl ChainID) Option {
 	}
 }
 
+func WithBlobs(blobs []*eth.Blob, config *params.ChainConfig) Option {
+	return func(tx *PlannedTx) {
+		tx.Type.Set(types.BlobTxType)
+		tx.BlobFeeCap.DependOn(&tx.AgainstBlock)
+		tx.BlobFeeCap.Fn(func(_ context.Context) (*uint256.Int, error) {
+			return uint256.MustFromBig(tx.AgainstBlock.Value().BlobBaseFee(config)), nil
+		})
+		var blobHashes []common.Hash
+		tx.Sidecar.Fn(func(_ context.Context) (*types.BlobTxSidecar, error) {
+			sidecar, hashes, err := txmgr.MakeSidecar(blobs, true)
```

Comment:
We're hardcoding the use of cell proofs / v1 sidecars here. Does that make sense?

Reply:
I suppose we could check for the Fusaka activation using `AgainstBlock`... Will do thanks.